package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "npm-conf",
-	"version": "1.1.2",
+	"version": "1.1.3",
```

Comment:
Don't bump version in a pull request. It's up to the maintainers when, how, and what version to bump.

Reply:
updated. thanks